docs/ref-manual/cli/options.md

```diff
@@ -399,6 +399,53 @@ Where `TokenPair` is
 We have two contracts `BankToken.sol` and `LoanToken.sol`. We want `tokenA` of the `tokenPair` to be `BankToken`, and `tokenB` to be `LoanToken`. Addresses take up only one slot. We assume `tokenPair` is the first field of Bank (so it starts at slot zero). To do that, we use:  
 `certoraRun Bank.sol BankToken.sol LoanToken.sol --verify Bank:Bank.spec --structLink Bank:0=BankToken Bank:1=LoanToken`
 
+Options for controlling contract creation
```

Comment:
running `make` returned 4 spelling errors; can you either correct them or add them to the dictionary?

Reply:
(actually, it looks like 3 of them were introduced in an earlier PR that I forgot to run spell checking on)

Reply:
(the dictionary is `spelling_wordlist.txt` in the root directory)